backend/src/modules/doubles-tournament/core/application/command/CallMatchCommandHandler.ts

```diff
@@ -0,0 +1,30 @@
+import { CommandHandler } from '../../../../../shared/core/application/command/CommandHandler';
+import { CallMatch } from './CallMatch';
+import { DomainEventPublisher } from '../../../../../shared/core/application/event/DomainEventBus';
+import { CommandResult } from '../../../../../shared/core/application/command/CommandResult';
+import { MatchWasCalled } from '../../domain/event/MatchWasCalled';
+import { CurrentTimeProvider } from '../../../../../shared/core/CurrentTimeProvider';
+import { CommandPublisher } from '../../../../../shared/core/application/command/CommandBus';
+import { BookTournamentTable } from '../../../../tournament-tables/core/application/command/BookTournamentTable';
+
+export class CallMatchCommandHandler implements CommandHandler<CallMatch> {
+  constructor(
+    private readonly eventPublisher: DomainEventPublisher,
+    private readonly currentTimeProvider: CurrentTimeProvider,
+    private readonly commandPublisher: CommandPublisher,
+  ) {}
+
+  async execute(command: CallMatch): Promise<CommandResult> {
+    const matchWasCalled = new MatchWasCalled({
```

Comment:
Czemu to jest tworzony juz obiekt, jak jeszcze nie wiadomo czy zdarzenie zajdzie :) ?

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/74/commits/0a4fd0754b111e4b38bf9a64534da80352f9e550
że tak wystarczy? 
Jeśli tak to faktycznie ma to sens :-)